### PR TITLE
Circle to gha

### PR DESCRIPTION
**Pull request by stevejalim**
TESTING

### CODE REVIEW

File: .github/workflows/pull_request_tests.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: Unit tests
+on: [pull_request]
```
Comment (pmac) [inline .github/workflows/pull_request_tests.yml:2]: should we also enable this for pushes to `main` (i.e. merges)?

Comment (stevejalim) [reply .github/workflows/pull_request_tests.yml:2]: Yeah good call. I'll check if circle runs under other situations too

**Review by pmac — APPROVED** [review]
This looks wonderful! Thanks for figuring this out! One question though, should we go ahead and remove the Circle config in this PR, or do that in a followup?

Either way :shipit:

**Comment by stevejalim** [comment]
Closing to make it against `mozilla/bedrock` instead

**Pull request closed without merging**